guides/v2.2/migration/migration-migrate-data.md

```diff
@@ -37,47 +39,21 @@ To force the Data Migration Tool to run from the beginning, use the `--reset` ar
 
 While running, the Data Migration Tool may report inconsistencies between Magento 1 and Magento 2 databases, and display messages like this:
```

Comment:
Need to change this mention to plural somehow since we introduce a set of examples

```suggestion
While running, the Data Migration Tool may report inconsistencies between Magento 1 and Magento 2 databases, and display messages like the following:
```

Reply:
**fixed**